package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Brandon Fulco <brandon.fulco@deque.com>",
   "license": "Unlicensed",
   "scripts": {
-    "lint": "eslint .",
+    "lint": "eslint ./src",
```

Comment:
Tested here and no more failures!
<img width="741" alt="Screenshot 2024-02-29 at 10 27 41 AM" src="https://github.com/dequelabs/action-a11y-issue-labeler/assets/3081483/45387833-066f-4d27-a982-4bfdea24bb6f">